spk/bazarr/Makefile

```diff
@@ -0,0 +1,41 @@
+SPK_NAME = bazarr
+
+include ../../cross/bazarr/bazarr.mk
```

Comment:
I would prefere to hardcode the version here (as we are used to) and not to include bazarr.mk containing the version.
It would be more obvious when the version is visible here - and on updates when you add the version to the changelog you can verify it matches within this Makefile.
We should define this consistently within the framework.
And may be we will find a framework solution to define all cross-package-versions in skp/Makefile. Until now we need two cross packages to support two different versions of the same package (see lirc and berkeleydb)